src/openforms/submissions/api/viewsets.py

```diff
@@ -166,7 +166,10 @@ def perform_create(self, serializer):
 
         # dispatch signal for modules to tap into
         submission_start.send(
-            sender=self.__class__, instance=serializer.instance, request=self.request
+            sender=self.__class__,
+            instance=serializer.instance,
+            request=self.request,
+            anonymous=serializer.validated_data.get("anonymous", False),
```

Comment:
maybe provide a default/initial on the serializer field so you can do `serializer.validated_data["anonymous"]`? That's a bit more straight-forward.